lib/semian/atomic_integer.rb

```diff
@@ -0,0 +1,20 @@
+module Semian
+  class AtomicInteger < SharedMemoryObject #:nodoc:
+    attr_accessor :value
+
+    def initialize(_name, _permissions)
+      @value = 0
+    end
+
+    def increase_by(val)
+      self.value += val
+    end
```

Comment:
Beside tests I don't see why we would increment by anything else than 1. So I would at least provide a shorthand that is `increment`